Oluwafemi John/plusMinus.py

```diff
@@ -0,0 +1,34 @@
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the plusMinus function below.
+def plusMinus(arr): 
```

Comment:
Space complexity wise, this isn't efficient. Something like this would be better

```py
def plusMinus(arr): 
    neg = 0
    pos = 0
    zer = 0

    for i in arr:
        if i < 0:
            neg += 1
        elif i == 0:
            zer += 1
        else:
            pos += 1

    a = pos/n
    b = neg/n
    c = zer/n
```

Reply:
Thanks..